### ISSUE TITLE
Still in development ?

### ISSUE THREAD

**Issue by dehlen** [i1skn/gamer#1]
Hi,

is this still in development ? In your last commit you added a "do not use in production" note. Could you elaborate what is missing/needs to be improved to remove this note ?

**Comment by i1skn** [comment]
Hi, yeah, it still in development. So it's not even ready to use now.
I did not have time quite a while to continue work on this. I plan to continue on this in near future, but if u have ideas or willing participate, thats will be cool!

**Comment by dehlen** [comment]
Yeah i would love to contribute since I need a turn based game server for a project I am working on and I'd love to use an already existing solution instead of starting from scratch. However it would be good to know why this project is not production ready. Which things are you planning to add, what is not working correctly of the already existing implementation. If you could give me a quick heads up on this I could look into it.

Cheers,
David

**Comment by i1skn** [comment]
Yeah, cool! So, I will refresh my memory about current state on the weekend and create document about what is missing, so we can continue on that! Sounds good for u?

Cheers,
Ivan

**Comment by dehlen** [comment]
Sounds perfect.
